exercicios/para-casa/entregas/michele-coelho/classes/InvoiceItem.js

```diff
@@ -0,0 +1,41 @@
+/*Escreva uma classe chamada InvoiceItem,
```

Comment:
Olá Michele, gostei muito do seu código, cumpriu o que o exercício gostaria de verdade.
Únicas observações são sobre
- O nome do método mensagem, além de estar diferente do proposto no exercício, é meio fora do padrão que costumamos colocar nos métodos, que geralmente são nomes de "ações". Então o melhor seria "MostrarMensagem" ou "EnviarMensagem" ou então mesmo o "toString" do exercício.
- Poderia também ter feito o método retornar diretamente a multiplicação no getTotal (return this.prince * this.quantity)

É isso, espero que faça sentido pra você as dicas, mas de qualquer forma seu código está bem resumido e ótimo, pois cumpre a proposta muito bem. :smiley:

Reply:
Oláá Ariel, faz sentido sim! Obrigada pelo comentário e pelas dicas , viu! <3